opentracing/span_standard_impl.go

```diff
@@ -0,0 +1,176 @@
+package opentracing
+
+import (
+	"fmt"
+	"reflect"
+	"sync"
+	"time"
+
+	"golang.org/x/net/context"
+)
+
+// Creates and returns a standard OpenTracer which defers to `rec` and
+// `ctxIdSource` as appropriate.
+//
+// See `SetGlobalTracer()`.
+func NewStandardTracer(rec Recorder, ctxIdSource ContextIDSource) OpenTracer {
```

Comment:
The stuff in this file would change as we make this package more robust... It's included here to illustrate some of the issues that come up, but is not as important as the `interface` definitions elsewhere in this package.